pkg/domain/infra/abi/containers_runlabel.go

```diff
@@ -178,13 +178,22 @@ func generateRunlabelCommand(runlabel string, img *image.Image, args []string, o
 }
 
 func replaceName(arg, name string) string {
+	if arg == "NAME" {
+		return name
+	}
+
 	newarg := strings.ReplaceAll(arg, "$NAME", name)
-	return strings.ReplaceAll(newarg, "${NAME}", name)
+	newarg = strings.ReplaceAll(newarg, "${NAME}", name)
+	return strings.ReplaceAll(newarg, "=NAME", fmt.Sprintf("=%s", name))
```

Comment:
could  `NAME` be the prefix of another variable `=NAMEFOO`?

Reply:
Theoretically yes.

Reply:
Fixed